lib/buildpackutil.py

```diff
@@ -327,8 +328,13 @@ def ensure_and_get_mono(mx_version, cache_dir):
         mono_location = _get_mono_path("/tmp/opt", mono_version)
     except NotFoundException:
         logging.debug("Mono not found in default locations")
+        linux_version = platform.linux_distribution()[2]
+        if linux_version == "bionic":
+          cdn_path = "/mx-buildserver/"
+        else:
+          cdn_path = "/mx-buildpack/"
```

Comment:
We discussed moving mono dependencies into the mx-buildpack folder to keep resources contained. We'll see those changes coming in...